lib/Option.cpp

```diff
@@ -251,6 +251,10 @@ static llvm::cl::opt<bool>
          llvm::cl::desc(
              "Enable support for FP64 (cl_khr_fp64 and/or __opencl_c_fp64)."));
 
+static llvm::cl::opt<bool> uniform_workgroup_size(
+    "uniform-workgroup-size", llvm::cl::init(false),
+    llvm::cl::desc("Disable support for non-uniform workgroup size."));
```

Comment:
A better description might be "Assume all workgroups are uniformly sized."

Reply:
Makes sense 👍